packages/snaps-controllers/src/services/offscreen/OffscreenExecutionService.ts

```diff
@@ -108,7 +108,7 @@ export class OffscreenExecutionService extends AbstractExecutionService<string>
 
     await chrome.offscreen.createDocument({
       justification: 'MetaMask Snaps Execution Environment',
-      reasons: ['IFRAME_SCRIPTING'],
+      reasons: ['IFRAME_SCRIPTING' as chrome.offscreen.Reason],
```

Comment:
Why is this needed?

Reply:
`Type '"IFRAME_SCRIPTING"' is not assignable to type 'Reason'.`